site/docs/v1.4/contributions/oracle-config.md

```diff
@@ -241,5 +241,5 @@ After creating the Velero server in your cluster, try this example:
 
 ## Additional Reading 
 
-* [Official Velero Documentation](https://velero.io/docs/v1.4.0-beta.1/)
+* [Official Velero Documentation](https://velero.io/docs/v1.4.0/)
```

Comment:
Any `v1.4.0` links need to change to `v1.4` with the new format.

Reply:
I think you found the other one :)